src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -260,7 +255,7 @@ public function setLoggers(array $loggers)
      *
      * @return callable|null The previous exception handler
      */
-    public function setExceptionHandler(callable $handler = null)
+    public function setExceptionHandler(?callable $handler): ?callable
```

Comment:
the param annotation is missing the null part. it can probably be removed though.